test/unit/package_test.rb

```diff
@@ -233,158 +233,158 @@ def test_centimetres_girth_cylinder
 
   end
 
-  def test_centimetres_girth
+  test "#centimetres_girth" do
```

Comment:
Similar to above, `#centimetres for girth of a non-cylinder`, or something along those lines.